java-server-side-ui/accelerator.yaml

```diff
@@ -0,0 +1,165 @@
+            name: includeAppSsoIntegration
+            value: true
+        - name: appSsoServerLabel
+          dependsOn:
+            name: includeAppSsoIntegration
+            value: true
+    - name: spring-boot-app-sso-auth-code-flow
+
+engine:
+  let:
+    - name: packageDirectory
+      expression: '#packageName.replace(".", "/")'
+    - name: workloadResourceName
+      expression: '#artifactId.toLowerCase()'
+  chain:
+    # Maven is selected
+    - condition: "#buildTool == 'maven'"
+      exclude: [ "settings.gradle.kts", "build.gradle.kts", "gradlew*", "gradle/**"]
+      merge:
+        - include: [ "**" ]
+          exclude: [ "pom.xml", "README.md" ]
+        - include: [ "pom.xml" ]
+          chain:
+            - type: ReplaceText
+              substitutions:
+                - text: "<groupId>com.vmware.tap.accelerators</groupId>"
+                  with: "'<groupId>' + #groupId + '</groupId>'"
+        - include: [ "README.md" ]
+          chain:
+            - type: ReplaceText
+              regex:
+                pattern: "--- StartGradle[\\s\\S]+?--- EndGradle"
+                with: "''"
+            - type: ReplaceText
+              regex:
+                pattern: "--- StartMaven\\s|--- EndMaven\\s"
+                with: "''"
+    # end of the Maven specific part
+
+    # Gradle is selected
+    - condition: "#buildTool == 'gradle'"
+      exclude: [ "pom.xml", "mvnw*", ".mvn/**"]
+      merge:
+        - include: [ "**" ]
+          exclude: [ "build.gradle.kts", "README.md" ]
+        - include: [ "build.gradle.kts" ]
+          chain:
+            - type: ReplaceText
+              substitutions:
+                - text: "group = \"com.vmware.tap.accelerators\""
+                  with: "'group = \"' + #groupId + '\"'"
+        - include: [ "README.md" ]
+          chain:
+            - type: ReplaceText
+              regex:
+                pattern: "--- StartMaven[\\s\\S]+?--- EndMaven"
+                with: "''"
+            - type: ReplaceText
+              regex:
+                pattern: "--- StartGradle\\s|--- EndGradle\\s"
+                with: "''"
+    # end of the Gradle specific part
+
+    - merge:
+      - include: [ "**" ]
+        exclude: [ "config/workload.yaml" ]
+      - include: [ "config/workload.yaml" ]
+        chain:
+          - type: ReplaceText
+            substitutions:
+              - text: "java-server-side-ui"
+                with: "#workloadResourceName"
+
+    - merge:
+      - type: InvokeFragment
+        reference: java-version
+      - include: [ "**" ]
+      onConflict: UseFirst
+
+    - condition: "#includeAppSsoIntegration"
+      merge:
+        - type: InvokeFragment
+          reference: app-sso-client
+          let:
+            - name: workloadName
+              expression: "#workloadResourceName"
+        - include: [ "**" ]
+      onConflict: UseFirst
+
+    - condition: "#includeAppSsoIntegration"
+      merge:
+        - type: InvokeFragment
```

Comment:
I don't see the placeholders in src/main/resources/application-local.yml replaced when generating, should I?

Reply:
That's correct. The local profile is for locally running the application. It is up to the developer to to fill in the values for a (shared) AppSSO instance. As locally no kubernetes spring cloud binding is available, a template would give some guidance (along with the documentation).